pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,75 @@
+
+- #anouncements: Important announcement from the JSON Schema organization.
+
+- #stack-overflow: Stack overflow questions tagged with JSON Schema.
+  
+## Best practices
+
+This is a list of best practices the JSON Schema project strives for and resources we recommend for additional information:
+
+1. File and branching naming conventions: We use dashes instead of spaces, and lowercase letters to name .md files and branches. For example: `implementers-guide.md`, `web-feat-diataxis`.
+2. Commit messages: To learn how to write good commit messages, see [Step-by-step guide on how to write good commit messages on Git](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/).
+
+## Contribution workflow 
+
+Here we describe three ways you can help us improve our documentation:
+
+
+1. Review pull requests
+2. Work on an existing issue 
+3. Propose changes 
+
+### Review pull requests
+
+Are you new to the JSON Schema community? Help us review pull requests. Reviewing pull requests helps you familiarize yourself with our workflows, learn how others contribute, and understand the project's needs and goals. It's a great way to start contributing and you'll be better prepared to make your own contributions.  
+
+### Work on an existing issue
+
+To work on an existing issue, go to the [JSON Schema docs board](https://github.com/orgs/json-schema-org/projects/16) and check all the open issues.  
+
+If the issue you select has no assignees, you can claim the issue by assigning it to yourself and tag @benjagm in the comments to let us know you'll be working on it. 
+
+If the issue you select has already an assignee, write a comment expressing your interest in collaborating with them. The JSON Schema respects the order in which people claim issues, therefore, you will have to check if the assignee is accepting collaborations.
+
+After you have been assigned to an issue, fork the [json-schema-org/website](https://github.com/json-schema-org/website) repository and create a new branch from `main` to work on the changes. 
+
+### Propose changes
+
+To propose modifications to our documentation that do not have an issue in the [documentation board](https://github.com/orgs/json-schema-org/projects/16), you can [create a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository) and use the *Documentation* template to describe the changes. After that, you can decide whether you want to work on the changes yourself or let someone else claim the issue. If you decide to work on the issue, assign it to yourself and commit the changes to a new branch. 
+
+## Create a Pull Request
+
+To submit your work to review by the community, open a draft pull request to the `main` upstream branch, and add the issue your pull request solves. Add @benjagm as a reviewer of your pull request, and let us know in the #documentation Slack channel your pull request is ready for review. 
```

Comment:
Instead asking for my review let's ask contributors to ask for "Docs Team" @json-schema-org/docs-team review.